tests/e2e.rs

```diff
@@ -1,30 +1,94 @@
 use assert_cmd::prelude::*;
 use regex::Regex;
 use std::fs;
-use std::process::Command;
+use std::process::{Child, Command};
 use std::thread;
 use std::time;
 
 mod utils;
 
+// ayb_cmd!("value1", value2; {
+//     "ENV_VAR" => env_value
+// })
+macro_rules! ayb_cmd {
+    ($($value:expr),+; { $($env_left:literal => $env_right:expr),* $(,)? }) => {
+        Command::cargo_bin("ayb")?
+                .args([$($value,)*])
+                $(.env($env_left, $env_right))*
+    }
+}
+
+// ayb_assert_cmd!("value1", value2; {
+//     "ENV_VAR" => env_value
+// })
+macro_rules! ayb_assert_cmd {
+    ($($value:expr),+; { $($env_left:literal => $env_right:expr),* $(,)? }) => {
+        Command::cargo_bin("ayb")?
+                .args([$($value,)*])
+                $(.env($env_left, $env_right))*
+                .assert()
```

Comment:
It might be overengineering or bad practice, but is it worth having `ayb_assert_cmd` call `ayb_cmd` and then to add the final assert/success calls?

Reply:
I could do that, but the tradeoffs (slower compilation times, potential lifetimes issues, etc) are not worth it